app/constants/currencies.js

```diff
@@ -0,0 +1,26 @@
+export const currencies = [
+    { value: "btc", label: "Bitcoin" },
+    { value: "eth", label: "Ethereum" },
+    { value: "ltc", label: "Litecoin" },
+    { value: "xrp", label: "Ripple" },
+    { value: "bch", label: "Bitcoin Cash" },
+    { value: "ada", label: "Cardano" },
+    { value: "xlm", label: "Stellar" },
+    { value: "dot", label: "Polkadot" },
+    { value: "link", label: "Chainlink" },
+    { value: "usdt", label: "Tether" },
+    { value: "doge", label: "Dogecoin" },
+    { value: "xmr", label: "Monero" },
+    { value: "uni", label: "Uniswap" },
+    { value: "wbtc", label: "Wrapped Bitcoin" },
+    { value: "vet", label: "VeChain" },
+    { value: "atom", label: "Cosmos" },
+    { value: "ltc", label: "Litecoin" },
+    { value: "bnb", label: "Binance Coin" },
+    { value: "vet", label: "VeChain" },
+    { value: "fil", label: "Filecoin" },
+    // Update crypto-currencies list if needed
+    { value: "not_specified", label: "Not Specified" },
```

Comment:
Later we have to copy cat a curated list of tokens per blockchain from https://github.com/trustwallet/assets/tree/master/blockchains for example.